WebHooks.msbuild

```diff
@@ -62,25 +75,26 @@
 
   <Target Name="RestorePackages" DependsOnTargets="DownloadNuGet">
     <Message Text="Restoring NuGet packages..." Importance="High" />
-    <Exec Command='"$(NuGetExe)" restore "%(SolutionsToBuild.Identity)" -PackagesDirectory packages -NonInteractive -Verbosity quiet -ConfigFile "$(MsBuildThisFileDirectory)\.nuget\NuGet.Config"' />
+    <Exec Command='SET BuildNumber=$(BuildNumber)
```

Comment:
Could these use the restore target? I think it also works on old-type csproj files now (if you have the relevant SDKs installed).

Reply:
Was thinking about that. I'll test it out&hellip;

If it works, do you want to see the new version before I merge into `dev`?

Reply:
(Not sure why this comment got hidden as outdated. I only moved an XML comment from a couple of lines down.)

Experiment failed. The `<restore/>` target supports only `<PackageReference />` and `<ProjectReference />` in the project file. https://docs.microsoft.com/en-us/nuget/schema/msbuild-targets#restore-target isn't very explicit about this but other sources are.